scripts/ES_requests.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2024, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import argparse
+import constant
+import socket
+import sys
+
+from functions.indexes.elasticsearch import get_eleasticsearch_host
+from functions.indexes.elasticsearch import check_status_eleasticsearch
+from functions.indexes.elasticsearch import request_elasticsearch
+from functions.plateform.plateform import get_plateform_info
+
+#
+# @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+#
+
+# Elastic REST API
+# cf. https://www.elastic.co/guide/en/elasticsearch/reference/current/rest-apis.html
```

Comment:
Add an example maybe?
ES_requests.py -m get -u /_cat/indices
to get all the indices ?